dotcom-rendering/src/layouts/lib/stickiness.tsx

```diff
@@ -36,7 +36,7 @@ const bannerWrapper = css`
 	position: fixed !important;
 	bottom: 0;
 	${getZIndexImportant('banner')}
-	max-height: 100vh;
+	max-height: 80vh;
```

Comment:
You probably want to [use the dynamic viewport height `dvh` if available](https://developer.mozilla.org/en-US/docs/Web/CSS/length#dynamic), as it will work better on mobile.

Reply:
👍 nice, I've added this as well